### PR TITLE
Create operation

### PR DESCRIPTION
**Pull request by mick-roper**
* added a `create-operation` action

### CODE REVIEW

File: create-operation/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Reliably integrates with GitHub as a [GitHub Action][gh-action] that you can add to your own GitHub CI/CD workflows.
+
+[gh-action]: https://github.com/features/actions
+[view-on-marketplace]: https://github.com/marketplace/actions/reliably-github-action
+
+You use the Reliably GitHub Action by including it in any of your GitHub workflow
+YAML files:
+
+```yaml
+name: Example workflow using Reliably
```
Comment (dmartin35) [inline create-operation/README.md:12]: maybe change the name of the example ?

File: create-operation/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+You can retrieve your access token once authenticated with the CLI:
+
+```console
+reliably auth status --show-token
+```
+
+## Specifying a custom folder for review
+
+```yaml
+name: Example workflow using Reliably with a custom folder
```
Comment (dmartin35) [inline create-operation/README.md:56]: not sure the name is valid ?

File: create-operation/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+on: push
+jobs:
+  demo:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout source code'
+        uses: actions/checkout@v2
+      - name: Get Reliably suggestions
```
Comment (dmartin35) [inline create-operation/README.md:64]: idem here

File: create-operation/example.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# A sample workflow which checks out the code for suggestions
+# using Reliably.
+# The results are then uploaded to GitHub Security Code Scanning.
+
+name: Reliably workflow
+on: push
+
+jobs:
+  k8s-code-scanning:
```
Comment (dmartin35) [inline create-operation/example.yaml:9]: not sure the name is correct here

File: create-operation/example.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# A sample workflow which checks out the code for suggestions
+# using Reliably.
+# The results are then uploaded to GitHub Security Code Scanning.
```
Comment (ciaranevans) [inline create-operation/example.yaml:3]: Not correct afaik

File: create-operation/example.yaml (hunk 4 above)
Comment (ciaranevans) [inline create-operation/example.yaml:9]: This isn't scanning

**Review by dmartin35 — CHANGES_REQUESTED** [review]
seems some of the names does not match your action

File: create-operation/README.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+  RELIABLY_TOKEN: ${{ secrets.RELIABLY_TOKEN }}
+```
+
+You can retrieve your access token once authenticated with the CLI:
+
+```console
+reliably auth status --show-token
+```
+
+## Specifying a custom folder for review
```
Comment (ciaranevans) [inline create-operation/README.md:53]: This example makes no mention of a custom folder - not sure it needs this example either

File: create-operation/README.md (hunk 3 above)
Comment (ciaranevans) [inline create-operation/README.md:64]: No suggestions being 'got' - this is creating an operation?

File: create-operation/README.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+      - name: 'Checkout source code'
+        uses: actions/checkout@v2
+      - name: Run Reliably scan
+        uses: reliablyhq/actions/operation@v1
+        env:
+          RELIABLY_TOKEN: ${{ secrets.RELIABLY_TOKEN }}
+          RELIABLY_ORG: ${{ secrets.RELIABLY_ORG }}
+```
+
+The code above adds a new job called `reliably` that checks out the code from your repository and then uses the Reliably GitHub Action to obtain any reliability advice and suggestions based on the code in the checked out repository.
```
Comment (ciaranevans) [inline create-operation/README.md:27]: I don't think this comment is correct

File: create-operation/README.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+```yaml
+name: Example workflow using Reliably
+on: push
+jobs:
+  reliably:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout source code'
+        uses: actions/checkout@v2
+      - name: Run Reliably scan
```
Comment (ciaranevans) [inline create-operation/README.md:20]: Probably not the correct name

**Review by ciaranevans — CHANGES_REQUESTED** [review]

File: create-operation/README.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+        env:
+          RELIABLY_TOKEN: ${{ secrets.RELIABLY_TOKEN }}
+          RELIABLY_ORG: ${{ secrets.RELIABLY_ORG }}
+        with:
+          service: my awesome service
+          type: build
+          outcome: success
+```
+
+The code above adds a new job called `reliably` that checks out the code from your repository and then uses the Reliably GitHub Action to obtain any reliability advice and suggestions based on the code in the checked out repository.
```
Comment (dmartin35) [inline create-operation/README.md:29]: I think this one is still not correct. only applies for K8s scanning

File: create-operation/README.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
+# Reliably GitHub Action
+
+Reliably integrates with GitHub as a [GitHub Action][gh-action] that you can add to your own GitHub CI/CD workflows.
+
+[gh-action]: https://github.com/features/actions
+[view-on-marketplace]: https://github.com/marketplace/actions/reliably-github-action
```
Comment (dmartin35) [inline create-operation/README.md:6]: this url has changed it seems, see 
https://github.com/marketplace/actions/reliably-actions

**Review by dmartin35 — CHANGES_REQUESTED** [review]

**Comment by dmartin35** [comment]
another global comment:
it could be nice as well , to update the main README, with a link to your action in subfolder, https://github.com/reliablyhq/actions/blob/master/README.md#available-actions

**Review by ciaranevans — APPROVED** [review]

**Review by dmartin35 — APPROVED** [review]
lgtm

**Pull request merged**